### DISCUSSION TITLE
Unable to install/load/set extensions on Kubernetes pod

### DISCUSSION THREAD

**Discussion by mohini-tripathi** [duckdb/duckdb#14027] [Q&A]
Hi Community,

Thank you so much for your support, I am new to duckdb and I have been facing an issue while running below code in a kubernetes pod shell, whereas the same code is working on my local, databricks environment.

`import duckdb
db = "my_database.duckdb"
duck_db_table = 'blob_properties'
con = duckdb.connect(db)
con.execute(f"""
    INSTALL azure;
    LOAD azure;
""")`
Installing azure part is stuck and keeps on running forever without any logs. I have tried duckdb CLI as well and faced the same issue. 
<img width="200" alt="Screenshot 2024-09-19 at 12 12 21 PM" src="https://github.com/user-attachments/assets/7b6d7b42-0f19-45c1-a365-52ea1e99477a">

Best Regards,
Mohini Tripathi

**Comment by carlopi** [comment]
Few informations would be useful, in particular the results for:
```sql
PRAGMA platform;
PRAGMA version;
```

A few things you can try are:
```sql
SET extension_directory = 'my_folder/extensions';
SET secret_directory = 'my_folder/stored_secrets';
```
This might help if the problem is in creating directories in the home folder (even though I would expect that to fail differently)

and/or, download and decompress extensions manually like:
```bash
wget https://extensions.duckdb.org/DUCKDB_VERSION/DUCKDB_PLATFORM/azure.duckdb_extension.gz
gzip -d azure.duckdb_extension.gz
```
using DUCKDB_VERSION  likely equal to `v1.1.0` (from your screenshot) and DUCKDB_PLATFORM to whatever `PRAGMA platform` returned.
Then duckdb side:
```sql
FORCE INSTALL 'path/to/just/unzipped/azure.duckdb_extension';
```
